web3/geth.py

```diff
@@ -68,253 +53,407 @@
+) -> Tuple[str, int, str, str]:
+    return (host, port, cors, apis)
 
-    def sign(self, message: str, account: ChecksumAddress, passphrase: str) -> HexStr:
-        return self._sign(message, account, passphrase)
 
-    def sign_typed_data(
-        self, message: Dict[str, Any], account: ChecksumAddress, passphrase: str
-    ) -> HexStr:
-        return self._sign_typed_data(message, account, passphrase)
+class GethAdmin(Module):
+    is_async = False
 
-    def unlock_account(
-        self, account: ChecksumAddress, passphrase: str, duration: Optional[int] = None
-    ) -> bool:
-        return self._unlock_account(account, passphrase, duration)
+    add_peer: Method[Callable[[EnodeURI], bool]] = Method(
+        RPC.admin_addPeer,
+        mungers=[default_root_munger],
+    )
 
+    datadir: Method[Callable[[], str]] = Method(
+        RPC.admin_datadir,
+        is_property=True,
+    )
 
-class AsyncGethPersonal(BaseGethPersonal):
-    is_async = True
+    node_info: Method[Callable[[], NodeInfo]] = Method(
+        RPC.admin_nodeInfo,
+        is_property=True,
+    )
 
-    async def ec_recover(
-        self, message: str, signature: HexStr
-    ) -> Awaitable[ChecksumAddress]:
-        return await self._ec_recover(message, signature)  # type: ignore
+    peers: Method[Callable[[], List[Peer]]] = Method(
+        RPC.admin_peers,
+        is_property=True,
+    )
 
-    async def import_raw_key(
-        self, private_key: str, passphrase: str
-    ) -> Awaitable[ChecksumAddress]:
-        return await self._import_raw_key(private_key, passphrase)  # type: ignore
+    start_http: Method[ServerConnection] = Method(
+        RPC.admin_startHTTP,
+        mungers=[admin_start_params_munger],
+    )
 
-    async def list_accounts(self) -> Awaitable[List[ChecksumAddress]]:
-        return await self._list_accounts()  # type: ignore
+    start_ws: Method[ServerConnection] = Method(
+        RPC.admin_startWS,
+        mungers=[admin_start_params_munger],
+    )
 
-    async def list_wallets(self) -> Awaitable[List[GethWallet]]:
-        return await self._list_wallets()  # type: ignore
+    stop_http: Method[Callable[[], bool]] = Method(
+        RPC.admin_stopHTTP,
+        is_property=True,
+    )
 
-    async def lock_account(self, account: ChecksumAddress) -> Awaitable[bool]:
-        return await self._lock_account(account)  # type: ignore
+    stop_ws: Method[Callable[[], bool]] = Method(
+        RPC.admin_stopWS,
+        is_property=True,
+    )
 
-    async def new_account(self, passphrase: str) -> Awaitable[ChecksumAddress]:
-        return await self._new_account(passphrase)  # type: ignore
 
-    async def send_transaction(
-        self, transaction: TxParams, passphrase: str
-    ) -> Awaitable[HexBytes]:
-        return await self._send_transaction(transaction, passphrase)  # type: ignore
+class GethMiner(Module):
+    """
+    https://github.com/ethereum/go-ethereum/wiki/Management-APIs#miner
```

Comment:
This goes to a redirect page, which then 404s. New link, or just remove?

Reply:
Good catch. Updated 👍🏼